__tests__/article.test.js

```diff
@@ -37,38 +36,157 @@ describe.only('/api/articles', () => {
       .expect(200)
       .then((response) => {
          const articles = response._body.articles;
-         articles.forEach((article) => {
-          expect(typeof article.author).toBe("string");
-          expect(typeof article.title).toBe("string");
-          expect(typeof article.topic).toBe("string");
-          expect(typeof article.votes).toBe("number");
-          expect(typeof article.article_img_url).toBe("string");
-          expect(typeof article.comment_count).toBe("string");
-         })
+         expect(articles.length).toEqual(13);
+         expect(articles).toMatchObject([
+          {
```

Comment:
rather than checking the entire value, we can still use a forEach and expect each object to match an object where the types are declared within them:  
expect(article).toMatchObject({
            article_id: expect.any(Number),
            title: expect.any(String),
            topic: expect.any(String),
            author: expect.any(String),
            created_at: expect.any(String),
            votes: expect.any(Number),
            article_img_url: expect.any(String),
        });